src/sdl/index.ts

```diff
@@ -141,6 +137,25 @@ export class SDL {
     }
   }
 
+  constructor(
+    public readonly data: BetaSdl<V>,
+    public readonly version: NetworkVersion = "beta2",
+    public readonly network: NetworkId = "mainnet"
```

Comment:
`network` is needed for some validation functions from cloudmos. E.g. denom validation